x-pack/plugins/profiling/server/index.ts

```diff
@@ -9,8 +9,19 @@ import { schema, TypeOf } from '@kbn/config-schema';
 import type { PluginConfigDescriptor, PluginInitializerContext } from '@kbn/core/server';
 import { ProfilingPlugin } from './plugin';
 
+const packageInputSchema = schema.object({
+  host: schema.string(),
```

Comment:
We should mark this optional too, we have a default config for it.
@florianl can you confirm that we do have a pre-defined `host` set?